src/v/redpanda/application.cc

```diff
@@ -100,6 +100,11 @@
 #include <exception>
 #include <vector>
 
+// This file in the data directory tracks the number of
+// unclean shutdowns.
+static constexpr std::string_view crash_loop_tracker_file
+  = "crash_loop_tracker";
```

Comment:
Since this file will exist in normal operation, not just when there's a problem, let's give it a less scary name, like "startup_log" (just a quick suggestion, I'm open to alternatives)